yellowbrick/text/dispersion.py

```diff
@@ -84,13 +88,25 @@ def fit(self, text):
             A list of words in the order they appear in the corpus.
         """
             
+        # Concatenating documents and define boundaries by word offset
+        self.boundaries_ = []
+        self.doc_words_ = []
+        self.offset = 0
```

Comment:
We don't really want to store the entire corpus in memory on the visualizer, it's possible that it's an extremely big corpus that's being stored on disk and a generator is reading through all the documents; which is why we had the generator test to ensure that we only looped through the input once (this loops through twice). 

Is it possible to compute the boundaries and the points in one single loop?